src/actions/submitUserInfoAction.js

```diff
@@ -0,0 +1,6 @@
+export const submitUserInfoAction = (infoObject = {}) => dispatch => {
+  dispatch({
+    type: "SUBMIT_USER_INFO_FORM_ACTION",
```

Comment:
These types should be defined as constants in `src/constants/actions.js`

Reply:
Done

Reply:
https://github.com/TiK15Ryyb/etasitsit-frontend/blob/2005f48ca6e5615d22a202fcf195cd6e2b30b6c8/src/components/UserForm.js#L18-L25

https://github.com/TiK15Ryyb/etasitsit-frontend/blob/2005f48ca6e5615d22a202fcf195cd6e2b30b6c8/src/components/UserForm.js#L52-L55

Reply:
https://github.com/TiK15Ryyb/etasitsit-frontend/blob/422af1aac61a235f67cebfdb4902afc6130929bd/src/actions/updateUserInfoFormAction.js#L1-L2

https://github.com/TiK15Ryyb/etasitsit-frontend/blob/422af1aac61a235f67cebfdb4902afc6130929bd/src/constants/actions.js#L1-L3